src/relay/backend/contrib/cmsisnn/relay_to_tir.cc

```diff
@@ -341,8 +347,8 @@ class RelayToTIRVisitor : public MixedModeMutator {
       clip_min = clip_attrs->a_min;
       clip_max = clip_attrs->a_max;
     } else {
-      clip_min = -128;
-      clip_max = 127;
+      clip_min = -static_cast<int>(1 << dtype_bits);
```

Comment:
Could you please also have a look at https://github.com/apache/tvm/blob/3680b3cb49f82a4dcff559ace509ed2ed3aa02da/src/relay/backend/contrib/cmsisnn/relay_to_tir.cc#L193? Looks incorrect there in the Conv2D implementation.